src/ui/util/columns/getCols.ts

```diff
@@ -0,0 +1,3107 @@
+import bySport from "../../../common/bySport";
```

Comment:
Is there some way to unify this with the other getCols? I don't like the duplicated info, it will inevitable fall out of sync by accident in the future.

Reply:
Merged the files probably needs more testing thou